src/mainboard/opentitan/crb/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "crb"
+version = "0.1.0"
+authors = ["Oreboot Authors"]
+edition = "2018"
+build = "build.rs"
+
+[dependencies]
+arch = { path = "../../../arch/riscv/rv32"}
+console = { path = "../../../console" }
+print = { path = "../../../lib/print" }
+model = { path = "../../../drivers/model" }
+payloads = { path = "../../../../payloads"}
+cpu = { path = "../../../cpu/lowrisc/ibex" }
```

Comment:
should be covered by the soc. 
board brings in soc
soc bring in cpu
cpu brings in arch
how we do it in coreboot and it's nice. This way, we have arch, soc, and cpu dependencies all over the place.

Reply:
Fixed! Except, we still need to import the arch to call "arch::halt()".

Reply:
The problem is that the soc might implement halt, via some soc-dependent mechanism, or the cpu, or the arch. So I would argue there should be a soc::halt, which might end up calling cpu::halt, which might just default down to arch::halt

or am I crazy